research-software-directory/RSD-as-a-service
# chore: upgrade to next v15.0.1

# Upgrade frontend to next v15.0.1

Changes proposed in this pull request:
* Upgrade next js from v14.2.3 to v15.0.1
* Upgrade eslint to v8.57.1 as the "oldest" supported eslint version in next v15

How to test:
* `make start` to build and generate test data
*  everything should work :-)

PR Checklist:
